src/makeRobot.js

```diff
@@ -38,7 +38,63 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, chip version: ${this.version}, `
+              + `wheels: ${this.wheels}`;
```

Comment:
pay attention to proper indentation
```suggestion
      return `name: ${this.name}, chip version: ${this.version}, `
        + `wheels: ${this.wheels}`;
```